### PR TITLE
Average interview time

### PR DESCRIPTION
**Pull request by jhayes45**
Stats route now has a new field "averageInterviewMinutes"

This field is the average number of minutes that people spoke to a recruiter for the given company, rounded up. (finishrecruiter - startrecruiter) This field only uses data from the past 24 hours.

This field returns -1 if nobody has completed talking to a recruiter yet.

The granularity of this field is company-specific, not individual recruiter-specific. To make it recruiter specific we would have to add a "recruiter_id" or similar field to LineEvents.

### CODE REVIEW

**Comment by jhayes45** [comment]
merge conflict fixed, stats based on midnight as requested

File: src/api/controllers/lines.js [hunk 1]
```diff
@@ -283,7 +290,8 @@ function getMyPlaceByEmployerId(user_id, employer_id) {
 function getAverageWaitTimeByEmployerId(employer_id) {
     //get all line events for this employer from the past 24 hours.
     //sort them by user_id, then by time of event.
-    var yesterday = new Date() - MS_IN_ONE_DAY;
+    var yesterday = new Date();
```
Comment (brianwang9100) [inline src/api/controllers/lines.js]: i think we should rename this variable to be more clear that it's midnight. rename to todaymidnight

Comment (brianwang9100) [reply src/api/controllers/lines.js]: oh call it, beginningOfDay or something

**Comment by jhayes45** [comment]
changed the variable name. merging

**Pull request merged**